<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_brewery</class>
 <widget class="QMainWindow" name="Qt_brewery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mashtun controller 6000</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonPID">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>PID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonHeater">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Heater</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPump">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Pump</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textSetTemp">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textSetP">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textSetI">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>75</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set P value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set I value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonConnect">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to contoller</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controler Status&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSerialBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="control_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="plotWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>721</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp_set">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Heater status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_heater_status">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time since start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_update">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dutycycle">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dutycycle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pump status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pump_status">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pvalue">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ivalue">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0.11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>I Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>85</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
